17thblog/singleBlogPageFrame.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+include('config/config.php');
+include('config/init.php');
+
+if($_REQUEST==null){
+	wrongPage();
+}
+
+//echo's normal page
+$post=getOneBlogPost($_REQUEST['blogPostID']);
+echoHeader("Blog Page", $post['title']);
+echoBlogPost($_REQUEST['blogPostID']);
+
+
+
+//comment form
+$Errors = array();
+
+if(isset($_REQUEST['commentForm'])){ //
+
+	validateFormField('Author');
+	validateFormField('Comment');
+
+	if(sizeof($Errors) == 0){
+		submitComment(
+			$_REQUEST['blogPostID'],
+			$_REQUEST['Author'],  //these have to match the form
+			$_REQUEST['Comment']
+		);
```

Comment:
As mentioned above, I'd always redirect after saving something to the db, even if you redirect back to the page. If you want to skip down to the comment that was just saved, look into using <a name='Comment_123' />

Although once this is moved into js (if you choose to do that) it'll probably not matter anymore.